installer/pkg/common/common.go

```diff
@@ -358,3 +360,30 @@ type TLS struct {
 	Certificate string `json:"cert"`
 	Key         string `json:"key"`
 }
+
+// validCookieChars contains all characters which may occur in an HTTP Cookie value (unicode \u0021 through \u007E),
+// without the characters , ; and / ... I did not find more details about permissible characters in RFC2965, so I took
+// this list of permissible chars from Wikipedia.
+//
+// The tokens we produce here (e.g. owner token or CLI API token) are likely placed in cookies or transmitted via HTTP.
+// To make the lifes of downstream users easier we'll try and play nice here w.r.t. to the characters used.
+var validCookieChars = []byte("0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz_-.")
+
+// RandomString produces a cryptographically secure random string of length N.
```

Comment:
Would it make sense to move this to something like `string_utils.go`? Probably other code snippets should also be moved to correctly named files.

This file is already 350+ lines and would likely keep growing. Maybe this request is out of this PRs scope but as the description says `Misc Improvements` you can consider this?

Reply:
A matter of taste I reckon. Usually I'd like to see a few more "string utlity" functions before introducing something like `string_utils.go`. Otherwise we'll quickly end up with a host of Go files each with one or two functions only.